### PR TITLE
Add bandage image module

### PR DESCRIPTION
**Pull request by heuermh**

### CODE REVIEW

File: tests/software/bandage/image/test.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- name: bandage image
+  command: nextflow run ./tests/software/bandage/image -entry test_bandage_image -c tests/config/nextflow.config
+  tags:
+    - bandage
+    - bandage_image
+  files:
+    - path: output/bandage/B-3106.png
+      should_exist: true
+    - path: output/bandage/B-3106.svg
+      should_exist: true
```
Comment (KevinMenden) [inline tests/software/bandage/image/test.yml:10]: You can omit the `should_exist` tag here, that's implied already by `path`
Did the same thing before :)

**Review by KevinMenden — COMMENTED** [review]
Looks good!
It looks like the tests didn't really run though. Not sure if that's currently disabled.
Should probably check that before merging.

File: software/bandage/image/meta.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: bandage_image
+name: bandage image
```
[the remark this answers is not in the document]
Comment (KevinMenden) [reply software/bandage/image/meta.yml:1]: hmm yes I think I confused it with the tests, where we aggred on having someting like `software tool` or similar.
It doesn't really change anything in the code and is not super important, but we should try to agree on something to make it lint-able. So just revert it back then :)

**Comment by KevinMenden** [comment]
Can you add the necessary lines to https://github.com/nf-core/modules/blob/master/.github/filters.yml? 
That should get the tests going.

**Review by KevinMenden — APPROVED** [review]
Thanks, tests passing, looking good! 👍

**Comment by heuermh** [comment]
Thank you, @KevinMenden @JoseEspinosa!

**Pull request merged**